### PR TITLE
Made changes to schedule route and added remindConfirm route back 

### PR DESCRIPTION
**Pull request by Barnes2197**

### CODE REVIEW

**Comment by Barnes2197** [comment]
#22

File: src/controllers/schedule.js [hunk 1]
```diff
@@ -48,16 +48,19 @@ const update = async (req, res) =>
...
 
         const schedule = await Schedule.findOneAndUpdate({title: title}, 
             {
                 title: title,
                 description: description,
                 host: host,
                 logo: logo,
-                category: category
+                category: category,
+                startTime: startTime,
```
Comment (Tommy2016x) [inline src/controllers/schedule.js]: key value pair assignments that are the same in an object can be written with just the name like so:
```
{
title,
description,
host,
...
}
```

File: src/controllers/schedule.js [hunk 2]
```diff
@@ -48,16 +48,19 @@ const update = async (req, res) =>
...
+                endTime: endTime
             }).exec();
+            
         httpResponse.successResponse(res, schedule);
     }catch(e)
     {
```
Comment (Tommy2016x) [inline src/controllers/schedule.js]: send a failure response if an error happens

Comment (Barnes2197) [reply src/controllers/schedule.js]: I have a  failure response on all the schedule functions

**Pull request merged**